cheap_repr/__init__.py

```diff
@@ -46,6 +49,23 @@
     __version__ = "???"
 
 
+if t.TYPE_CHECKING:
+    from collections import abc
+
+    T = t.TypeVar('T')
+    try:
+        P = t.ParamSpec('P')
+    except AttributeError:
```

Comment:
This will never be raised since this doesn't happen at runtime. Is this kind of logic something most type checkers handle well?